Semantic Scholar.js

```diff
@@ -99,16 +100,16 @@ function parseDocument(doc, url) {
 			mimeType: 'application/pdf'
 		});
 	}
-	else {
-		item.attachments.push({
-			url: paperLink,
-			title: "Publisher Link",
-			mimeType: "text/html",
-			snapshot: false
-		});
-	}
 
-	item.complete();
+	// use the public api to retrieve more structured data
+	const paperIdRegex = /\/(.{40})(\?|$)/;
+	const paperId = paperIdRegex.exec(url)[1];
+	const apiUrl = `http://api.semanticscholar.org/v1/paper/${paperId}?client=zotero_connect`;
```

Comment:
This should be HTTPS.

Reply:
Sure agree that would be better (looks like the api docs need to get updated to list https support), fixed.